src/js/dataFields.js

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2016 Michael Warren Skirpan
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+*/
+
+(function (d3, $, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.dataDashboard.graphCanvas.dataFields", {
+        gradeNames: ["fluid.viewComponent"],
+        selectors: {
+            fieldTiles: ".floec-dataFields-tiles"
+        },
+        listeners: {
+            "{floe.dataDashboard.graphCanvas}.events.dataParsed" : {
+                funcName: "floe.dataDashboard.dataFields.createTiles",
+                args: ["{dataFields}", "{arguments}.0"]
+            }
+        },
+        model: {
+            fields: [],
+            state: "{floe.dataDashboard.graphCanvas}.model.selectionState"
+        }
+
+    });
+
+    floe.dataDashbaord.dataFields.createTiles = function (that, fields) {
+        
+        tilesDiv = that.locate("fieldTiles");
+        tilesDiv.append("<ol class='floec-dataFields-tiles'>");
+        fluid.each(fields, function (val, index){
+            var nextField = "<li value=" + val + ">" + val + "</li>";
```

Comment:
Pull out this markup into a component options block in, say, 
markup: {
    field: "<li value=\"%value\">%value</li>",
    fields: "<ol class=\"floec-dataFields-tiles\">%fields</ol>"
}
And then the markup can be rendered by two uses of http://docs.fluidproject.org/infusion/development/CoreAPI.html#fluid-stringtemplate-template-terms- 

This would be a useful intermediate point before moving towards something more substantial like, say, handlebars, if it was required.

For binding of the click event you can use a jQuery delegation system such as http://api.jquery.com/on/#direct-and-delegated-events to avoid constructing one listener per element